### DISCUSSION TITLE
Google SAML: Which data are shared with Google?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#4731] [Questions & Answers] [answered]
Is there any user information exchanged between kimai and Google (esp. from kimai to Google) beyond name, title, account no, email address, and groups?

Google knows when a user logs into kimai, but apart from that it will not learn any addition user information or attributes?

**Comment by kevinpapst** [comment, marked as answer]
Google passes information to Kimai, not the other way around.

**Reply by EduNetEurope** [reply]
Thanks!
